src/Common/MetricHelper.cs

```diff
@@ -50,13 +50,13 @@ public class MetricHelper
         [ThreadStatic]
         private static string _internalCalledCmdlets = "";
```

Comment:
Please remove the field initialization to string type as by default it should be null in order to avoid any confusion when some one unfamiliar with this attribute reads the code in the future.